users/luescher/setups/librispeech/pipeline_base_args.py

```diff
@@ -322,25 +336,25 @@ def get_triphone_args(
     triphone_recognition_args = {
         "iters": [8, 10],
         "feature_flow": feature_flow,
-        "pronunciation_scales": [6.0],
-        "lm_scales": [24.9],
+        "pronunciation_scales": [1.0],
+        "lm_scales": [25.0],
         "lm_lookahead": True,
         "lookahead_options": None,
         "create_lattice": True,
         "eval_single_best": True,
         "eval_best_in_lattice": True,
         "search_parameters": {
-            "beam_pruning": 12.0,
-            "beam-pruning-limit": 100000,
+            "beam_pruning": 15.0,
+            "beam-pruning-limit": 50000,
             "word-end-pruning": 0.5,
-            "word-end-pruning-limit": 15000,
+            "word-end-pruning-limit": 10000,
```

Comment:
10k here is a good idea though, for a dev-other recognition I had about 20-30 sequences above 10k and the rest below.